terraform/environments/digital-prison-reporting/application_variables.json

```diff
@@ -14,9 +14,15 @@
       "reporting_hub_batch_duration_seconds": 30,
       "reporting_hub_add_idle_time_between_reads": true,
       "reporting_hub_idle_time_between_reads_in_millis": 140,
+      "reporting_hub_retry_max_attempts": 10,
+      "reporting_hub_retry_min_wait_millis": 100,
+      "reporting_hub_retry_max_wait_millis": 10000,
```

Comment:
The time spent waiting on each iteration with these setting is roughly like this (not including the time it takes to actually run the operations):
```
Retrying after attempt 1. Total time spent waiting: 204ms.
Retrying after attempt 2. Total time spent waiting: 442ms.
Retrying after attempt 3. Total time spent waiting: 1,013ms.
Retrying after attempt 4. Total time spent waiting: 2,060ms.
Retrying after attempt 5. Total time spent waiting: 3,948ms.
Retrying after attempt 6. Total time spent waiting: 6,641ms.
Retrying after attempt 7. Total time spent waiting: 13,488ms.
Retrying after attempt 8. Total time spent waiting: 23,605ms.
Retrying after attempt 9. Total time spent waiting: 31,772ms.
Retries exceeded on attempt 10. Total time spent waiting: 31,774ms.
```